.github/workflows/try-state.yml

```diff
@@ -0,0 +1,62 @@
+# Runs `try-state` on all live runtimes in `polkadot-sdk`
+
+name: Check try-state
+on:
+  push:
+    branches:
+      - liam-try-state-scheduled-checks
+  schedule:
+    # run once per day at midnight
+    - cron: "0 0 * * *"
+
+env:
+  # TODO: Replace this with paritytech/polkadot-sdk 'latest' release URL once 1.3.0 is published 
```

Comment:
Can do now?

Reply:
1.3.0 did not include the Github workflow that builds the runtimes so we need to wait 1 more release for that to happen automatically, but I'm inclined to upload the debug builds there manually so we can use it.